Just tried to build gdal with this PR and it fails on following tests:

```
=================================== FAILURES ===================================
________________ test_check_geometry_equals_orientation_differs ________________

    def test_check_geometry_equals_orientation_differs():
        poly_ccw = ogr.CreateGeometryFromWkt("POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))")
        poly_cw = ogr.CreateGeometryFromWkt("POLYGON ((0 0, 0 1, 1 1, 0 1, 0 0))")

        if ogrtest.have_geos():
>           ogrtest.check_feature_geometry(poly_ccw, poly_cw)

/build/source/build/autotest/ogr/ogr_ogrtest.py:93:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/nix/store/h3i0acpmr8mrjx07519xxmidv8mpax4y-python3-3.12.5/lib/python3.12/contextlib.py:81: in inner
    return func(*args, **kwds)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

args = (<osgeo.ogr.Geometry; proxy of <Swig Object of type 'OGRGeometryShadow *' at 0x7ffedad47870> >, <osgeo.ogr.Geometry; proxy of <Swig Object of type 'OGRGeometryShadow *' at 0x7ffedad459e0> >, 0.0001)
kwds = {'_root_actual': <osgeo.ogr.Geometry; proxy of <Swig Object of type 'OGRGeometryShadow *' at 0x7ffedad45740> >, '_root...ected': <osgeo.ogr.Geometry; proxy of <Swig Object of type 'OGRGeometryShadow *' at 0x7ffedad45ce0> >, 'context': None}

    @wraps(func)
    def inner(*args, **kwds):
        with self._recreate_cm():
>           return func(*args, **kwds)
E           AssertionError: Error in vertex 2/5 exceeds tolerance.
E               Expected: POLYGON ((0 0,0 1,1 1,0 1,0 0))
E               Actual: POLYGON ((0 0,1 0,1 1,0 1,0 0))
E           assert [1.0, 0.0] == approx([0.0 ±....0 ± 1.0e-04])
E
E             comparison failed. Mismatched elements: 2 / 2:
E             Max absolute difference: 1.0
E             Max relative difference: inf
E             Index | Obtained | Expected
E             0     | 1.0      | 0.0 ± 1.0e-04
E             1     | 0.0      | 1.0 ± 1.0e-04

/nix/store/h3i0acpmr8mrjx07519xxmidv8mpax4y-python3-3.12.5/lib/python3.12/contextlib.py:81: AssertionError
=========================== short test summary info ============================
FAILED ogr/ogr_ogrtest.py::test_check_geometry_equals_orientation_differs - AssertionError: Error in vertex 2/5 exceeds tolerance.
    Expected: POLYGON ((0 0,0 1,1 1,0 1,0 0))
    Actual: POLYGON ((0 0,1 0,1 1,0 1,0 0))
assert [1.0, 0.0] == approx([0.0 ±....0 ± 1.0e-04])

  comparison failed. Mismatched elements: 2 / 2:
  Max absolute difference: 1.0
  Max relative difference: inf
  Index | Obtained | Expected
  0     | 1.0      | 0.0 ± 1.0e-04
  1     | 0.0      | 1.0 ± 1.0e-04
= 1 failed, 12790 passed, 1418 skipped, 6 deselected, 6 xfailed in 246.52s (0:04:06) =
100 - done.
error: builder for '/nix/store/7ngzx765j6kizhwx75rip27z18ifk0j7-gdal-3.9.2.drv' failed with exit code 1;
       last 10 log lines:
       > assert [1.0, 0.0] == approx([0.0 ±....0 ± 1.0e-04])
       >
       >   comparison failed. Mismatched elements: 2 / 2:
       >   Max absolute difference: 1.0
       >   Max relative difference: inf
       >   Index | Obtained | Expected
       >   0     | 1.0      | 0.0 ± 1.0e-04
       >   1     | 0.0      | 1.0 ± 1.0e-04
       > = 1 failed, 12790 passed, 1418 skipped, 6 deselected, 6 xfailed in 246.52s (0:04:06) =
       > 100 - done.
       For full logs, run 'nix log /nix/store/7ngzx765j6kizhwx75rip27z18ifk0j7-gdal-3.9.2.drv'.
```

--- a/pkgs/development/libraries/gdal/default.nix
+++ b/pkgs/development/libraries/gdal/default.nix
@@ -2,6 +2,7 @@
 , stdenv
 , callPackage
 , fetchFromGitHub
+, fetchpatch
 
 , useMinimalFeatures ? false
 , useTiledb ? (!useMinimalFeatures) && !(stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64)
@@ -87,6 +88,13 @@ stdenv.mkDerivation (finalAttrs: {
     rev = "v${finalAttrs.version}";
     hash = "sha256-BXnpNfi9tUd6nnwYdstuOfGsFVif8kkmkW97X1UAgt8=";
   };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/OSGeo/gdal/commit/91e4f55f8f374a75f8f2ecd05670edcfa4c0af84.patch";
+      sha256 = "sha256-C2lkZLsORso7WVxgX79r5swkoVu/APPwQp2C/rmmCAo=";
+    })
+  ];
 
   nativeBuildInputs = [
     bison

--- a/pkgs/development/libraries/geos/default.nix
+++ b/pkgs/development/libraries/geos/default.nix
@@ -9,11 +9,11 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "geos";
-  version = "3.12.2";
+  version = "3.13.0";
 
   src = fetchurl {
     url = "https://download.osgeo.org/geos/geos-${finalAttrs.version}.tar.bz2";
-    hash = "sha256-NMd3C/AJDuiEiK+Ydn0I53nxJPozQ34Kq+yKvUYJ/sY=";
+    hash = "sha256-R+yD/zNNZyueRCZpXxXabmNoJEIUlx+r84b/jvbfOeQ=";
   };
 
   nativeBuildInputs = [ cmake ];

--- a/pkgs/development/libraries/librasterlite2/default.nix
+++ b/pkgs/development/libraries/librasterlite2/default.nix
@@ -63,7 +63,7 @@ stdenv.mkDerivation rec {
     libspatialite
     libtiff
     libwebp
-    libxml2
+    (libxml2.override { enableHttp = true; })
     lz4
     minizip
     openjpeg
@@ -71,7 +71,7 @@ stdenv.mkDerivation rec {
     proj
     sqlite
     zstd
-  ] ++ lib.optional stdenv.hostPlatform.isDarwin ApplicationServices;
+  ] ++ lib.optional stdenv.isDarwin ApplicationServices;
 
   enableParallelBuilding = true;
 

